xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -525,3 +525,14 @@
 #macro(glyphicon $iconName)
 #if("$!iconName"!='')<span class="glyphicon glyphicon-${iconName}"></span>#end
 #end
+#macro(refactoringStatusHeader $stepTitle $titleHint)
```

Comment:
#refactoringStatusHeader sounds a bit strange. 
First because, imagine in 3 years, someone will want to use the macro, will they be doing a refactoring then? 

The macro mostly is creating the page title, and is not used only on status pages.

Reply:
There is a template named contentheader.vm.
Should we make it `headerTitle` instead?

Reply:
`#document-header` or `#document-title` ? @mflorea do we have any standards / patterns for macro naming?

Reply:
#document-header

Reply:
I notice that there are no hyphens in any of the macro names.
I will make it `#documentTitle` if thats fine with you.
And I tried looking up the code style but the page for 'Velocity Macro' doesnt exist.

Reply:
Yea, apparently it doesn't exists https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/

Reply:
@evalica macro name should use camel case and it usually starts with a verb (like you would name a method in Java).

Reply:
I think #showDocumentTitle is a good choice.